NEWS

```diff
@@ -156,6 +156,10 @@ Changed Functionality
 - `detect-protocol.zeek was the last non-deprecated policy script left in
   `frameworks/dpd`.  It was moved to `frameworks/analyzer/detect-protocol.zeek`.
 
+- The `not_valid_before` and `not_valid_after` times are now logged as GMT timestamps.
+  Before, the times were logged as local times; thus the output was dependent on the
+  timezone that your system is set to.
```

Comment:
Hmm, I guess we actually parsed them into the wrong timezone? That is, already the timestamp fields on the `X509::Certificate` field were off. I'm not sure we need to talk about that, but maybe there's someone out there using `Certificate$not_valid_before` directly and they'll now see a behavior change.

Reply:
If the local timezone was not GMT, we did; I updates the NEWS entry. If this looks good to you I will merge this to master.